app/stacks/uk-west/back-office/database.tf

```diff
@@ -0,0 +1,79 @@
+resource "random_password" "back_office_sql_server_password" {
+  length      = 32
+  special     = true
+  min_lower   = 2
+  min_upper   = 2
+  min_numeric = 2
+  min_special = 2
+}
+
+resource "random_id" "username_suffix" {
+  byte_length = 6
+}
+
+# tflint-ignore: terraform_unused_declarations

```

Comment:
Can you remove these?

Reply:
Fixed in https://github.com/Planning-Inspectorate/infrastructure-environments/pull/60/commits/8bf0fe49566d10f7b16de2f782ceea6dc0dafd83